File: sdks/ruby/lib/novabilling/credit_notes/types/create_credit_note_dto_reason.rb
# frozen_string_literal: true

module Novabilling
  module CreditNotes
    module Types
      module CreateCreditNoteDtoReason
        extend Novabilling::Internal::Types::Enum

        DUPLICATE = "DUPLICATE"
        PRODUCT_UNSATISFACTORY = "PRODUCT_UNSATISFACTORY"
        ORDER_CHANGE = "ORDER_CHANGE"
        OTHER = "OTHER"
      end
    end
  end
end

File: sdks/php/src/CreditNotes/Types/CreateCreditNoteDtoReason.php
<?php

namespace NovaBilling\CreditNotes\Types;

enum CreateCreditNoteDtoReason: string
{
    case Duplicate = "DUPLICATE";
    case ProductUnsatisfactory = "PRODUCT_UNSATISFACTORY";
    case OrderChange = "ORDER_CHANGE";
    case Other = "OTHER";
}
